integration/consul_catalog_test.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"io/ioutil"
+	"net/http"
+	"os/exec"
+	"time"
+
+	"github.com/hashicorp/consul/api"
+	checker "github.com/vdemeester/shakers"
+	check "gopkg.in/check.v1"
+)
+
+// Consul catalog test suites
+type ConsulCatalogSuite struct {
+	BaseSuite
+	client *api.Client
+}
+
+func (s *ConsulCatalogSuite) SetUpSuite(c *check.C) {
+	s.createComposeProject(c, "consul_catalog")
+	s.composeProject.Up()
+
+	config := api.DefaultConfig()
+	config.Address = "127.0.0.1:8500"
```

Comment:
Thanks for looking into this :)

Because everything uses mapped ports it should be possible to point to the real host here but perhaps it would be cleaner to find the ip of the instances I need through listing the containers through the api. Is something like that what you think it should look like?